### DISCUSSION TITLE
Home Assistant Sensors for Inverter and Strings hold latest state (MQTT)

### DISCUSSION THREAD

**Discussion by mime24** [s-allius/tsun-gen3-proxy#233] [General]
If my inverter is off at nights the Home Assistant Sensors hold their latest status. 
I would prefer the sensors become ‘unavailable’ if the connection is dead. 

Any thought on this?

**Comment by s-allius** [comment]
Hi Michael, good idea. I've never thought about the availability switch and how we can or should use it for the proxy. Sounds logical though.
I would make a note of this topic for the next version
